PyPI_analysis/main.py

```diff
@@ -292,6 +359,14 @@ def main(
         return 5
 
     exit_code = assign_exit_code(analysis=analysis)
-    print_output(analysis=analysis, stdout=stdout)
+
+    log_contents = log_buffer.getvalue()
+    warning_files = {
```

Comment:
I think we could change the name of this to `warnings_entries` or `warnigs_limited_eval`. Current name suggests that there are files involved.

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/404/commits/b07989f82b3924ec305dc5064d3e1be9e0bdcf6b